src/domains/landing/components/LoginForm.tsx

```diff
@@ -0,0 +1,113 @@
+import { ThemeOverride } from '@emotion/react';
+import { unwrapResult } from '@reduxjs/toolkit';
+import { useFormik } from 'formik';
+import React, { FC, useRef } from 'react';
+import { TextInput, TextStyle, View, ViewStyle } from 'react-native';
+import { useDispatch } from 'react-redux';
+import { Button } from 'src/components/Button';
+import { Input } from 'src/components/Input';
+import { PasswordInput } from 'src/components/PasswordInput';
+import { Text } from 'src/components/Text';
+import { AppDispatch } from 'src/state';
+import { login } from 'src/state/authSlice';
+import * as Yup from 'yup';
+
+const VALIDATION_SCHEMA = Yup.object({
+  email: Yup.string().email('Email is invalid').required('Email is required'),
+  password: Yup.string().max(255, 'Password too long').required('Password is required'),
```

Comment:
i like yup's format

Reply:
password min length?

Reply:
Technically I should probably remove the max, doesn't make sense to restrict this on login (just on signup)